### PR TITLE
Fixed up the last iTable for each class

### PR DESCRIPTION
**Pull request by Ershany-IBM**

### CODE REVIEW

**Review by akshayben — APPROVED** [review]

File: runtime/vm/JVMImage.cpp [hunk 1]
```diff
@@ -109,6 +109,11 @@ JVMImage::setupColdRun(void)
 		return IMAGE_ERROR;
 	}
 
+	_invalidTable = (J9ITable *) subAllocateMemory(sizeof(J9ITable));
```
Comment (akshayben) [inline runtime/vm/JVMImage.cpp]: maybe also make _invalidTable->next = NULL;

Comment (akshayben) [reply runtime/vm/JVMImage.cpp]: This might be wrong I haven't checked the struct

Comment (Ershany-IBM) [reply runtime/vm/JVMImage.cpp]: I will take a look at this tomorrow

**Comment by akshayben** [comment]
`const J9ITable invalidITable = { (J9Class *) (UDATA) 0xDEADBEEF, 0, (J9ITable *) NULL };`

this is in hshelp.c. I think you could just memset the Itable to 0. The class reference might be also just a dummy class

**Pull request merged**